src/demo-app/a11y/datepicker/datepicker-a11y.html

```diff
@@ -0,0 +1,112 @@
+<section>
+  <h2>Choose a date (e.g. choose your date of birth)</h2>
+  <md-input-container>
+    <input mdInput
```

Comment:
This input needs either a `<label>` or an `aria-label`
(same for other inputs that don't have one)